docs/resources/sfs_turbo_du_task.md

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Scalable File Service (SFS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_sfs_turbo_du_task"
+description: ""
+---
+
+# huaweicloud_sfs_turbo_du_task
+
+Manages a DU task resource under the SFS Turbo within HuaweiCloud.
+
+-> SFS Turbo File systems created after August 1, 2023 support this resource.
+    If there are `10` executing tasks, no new tasks areallowed to be created.
+    It is recommended that no more than `4` query requests be submitted at the same time.
+    Querying a very large directory tasks a long time, please do not submit the request again.
+
+-> The resource supported SFS Turbo file system types are **20MB/s/TiB**, **40MB/s/TiB**, **125MB/s/TiB**,
```

Comment:
-> This resource is only available for the following SFS Turbo file system types: XXX